packages/patternfly-4/react-charts/src/components/ChartDonut/ChartDonut.js

```diff
@@ -1,21 +1,130 @@
 
 export const propTypes = {
   /**
    * See TypeScript API docs: https://formidable.com/open-source/victory/docs/victory-area/
    */
-  '': PropTypes.any
+  '': PropTypes.any,
+  /**
+   * The height props specifies the height the svg viewBox of the chart container. This value should be given as a
+   * number of pixels.
+   *
+   * Because Victory renders responsive containers, the width and height props do not determine the width and
+   * height of the chart in number of pixels, but instead define an aspect ratio for the chart. The exact number of
+   * pixels will depend on the size of the container the chart is rendered into.
+   *
+   * Note: innerRadius may need to be set when using this property.
+   */
+  height: PropTypes.number,
+  /**
+   * The subtitle for the donut chart
+   */
+  subTitle: PropTypes.string,
+  /*
+   * Specifies the theme color; blue (default), green, or multi-color. Overridden by the theme property.
+   */
+  themeColor: PropTypes.string,
+  /*
+   * Specifies the theme variant; 'dark' or 'light' (default). Overridden by the theme property.
+   */
+  themeVariant: PropTypes.string,
+  /**
+   * The title for the donut chart
+   */
+  title: PropTypes.string,
+  /**
+   * The width props specifies the width of the svg viewBox of the chart container. This value should be given as a
+   * number of pixels.
+   *
+   * Because Victory renders responsive containers, the width and height props do not determine the width and
+   * height of the chart in number of pixels, but instead define an aspect ratio for the chart. The exact number of
+   * pixels will depend on the size of the container the chart is rendered into.
+   *
+   * Note: innerRadius may need to be set when using this property.
+   */
+  width: PropTypes.number
 };
 
+class ChartDonut extends React.Component {
+  childeRef = React.createRef();
+
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      height: this.props.height || 0,
+      width: this.props.width || 0
+    };
+  }
+
+  componentDidMount() {
+    // Skip resize handler if height and width were provided
+    if (!(this.state.height || this.state.width)) {
+      setTimeout(() => {
+        window.addEventListener('resize', this.handleResize);
+      });
+      this.handleResize();
+    }
+  }
+
+  componentWillUnmount() {
+    window.removeEventListener('resize', this.handleResize);
+  }
+
+  handleResize = () => {
+    if (this.childeRef.current && this.childeRef.current.parentNode) {
+      this.setState({
+        height: this.childeRef.current.parentNode.clientHeight,
+        width: this.childeRef.current.parentNode.clientWidth
+      });
+    }
+  };
+
+  render() {
+    const { subTitle, theme, themeColor, themeVariant, title, ...rest } = this.props;
+    const { height, width } = this.state;
+
+    const chartTheme = theme || getDonutTheme(themeColor, themeVariant);
+    const showLabels = subTitle || title;
```

Comment:
I don't think we need this. If no title or subtitle exists, then the ChartLabel component won't render anything.

Reply:
If there is no title or subtitle, there is no need to wrap with VictoryContainer.

Reply:
Right, but if the user does provide labels, and sets `standalone === false`, this breaks.